c10/core/ScalarType.h

```diff
@@ -272,6 +274,10 @@ static inline bool isQIntType(ScalarType t) {
       t == ScalarType::QUInt2x4;
 }
 
+static inline bool isBitType(ScalarType t) {
```

Comment:
isBitsType? Also not entirely clear if you really want this function, we'll have to see I guess